packages/web/src/js/Modal.ts

```diff
@@ -1,25 +1,30 @@
 import BaseComponent from './BaseComponent';
 import EventHandler from './dom/EventHandler';
 import SelectorEngine from './dom/SelectorEngine';
-import { getElementFromSelector, reflow } from './utils';
+import { enableToggleTrigger } from './utils/ComponentFunctions';
+
+const NAME = 'modal';
 
 const MODAL_TOGGLE_SELECTOR = '[data-toggle="modal"]';
-const MODAL_DISMISS_ATTRIBUTE = 'data-dismiss';
-const BACKDROP_TAG_NAME = 'div';
-const BACKDROP_CLASSNAME = 'Modal__backdrop';
+// const BACKDROP_TAG_NAME = 'div';
+// const BACKDROP_CLASSNAME = 'Modal__backdrop';
 
 class Modal extends BaseComponent {
   isShown: boolean;
-  backdrop: HTMLElement;
+  // backdrop: HTMLElement;
```

Comment:
@adamkudrna Because the dialog element was used before this new composition of the Modal component, we can delete the old one, I think.

Reply:
We can safely remove it.

Reply:
🤞 Removed.